src/pages/scale.tsx

```diff
@@ -102,117 +102,146 @@ export function Scale({
             />
           </ButtonGroup>
         </Box>
-        <ZStack>
-          <Box
-            sx={{
-              display: 'flex',
-              width: '100%',
-              height: '100%',
-              borderRadius: 2
+        <div
+          style={{
+            flexGrow: 1,
+            display: 'grid',
+            gridTemplateRows: 'auto 1fr',
+            gap: 8
+          }}
+        >
```

Comment:
If you use Primer's `Box` component with the `sx` prop you can use design system values instead of pixel values: 
```suggestion
        <Box
          sx={{
            flexGrow: 1,
            display: 'grid',
            gridTemplateRows: 'auto 1fr',
            gap: 2
          }}
        >
```